src/magellan_firmware/firmware/magellan_controller/IMU.h

```diff
@@ -20,7 +20,7 @@ class IMU {
     sensor_msgs::Imu imu_msg_;
     magellan_core::IMUState imu_state_msg_;
     adafruit_bno055_offsets_t sensor_offsets_;
-    // IMU constants
+// IMU constants
```

Comment:
why does this get unindented?

Reply:
clang-tidy wasn't looking at headers, this was done by another linter.